Casks/8/8bitdo-ultimate-software-v2.rb

```diff
@@ -0,0 +1,23 @@
+cask "8bitdo-ultimate-software-v2" do
+  version "1.0.11"
+  sha256 "a06ed63f5d5917f884e72c41aefbacd829b1fb16646bdcbf448539bf2c185342"
+
+  url "https://download.8bitdo.com/Ultimate-Software/8BitDo_Ultimate_Software_V2_macOS_V#{version}.zip"
+  name "8bitdo-ultimate-software-v2"
+  desc "Control every piece of your controller"
+  homepage "https://app.8bitdo.com/Ultimate-Software-V2/"
+
+  livecheck do
+    url :url
+    regex(/^macOS[._-]V?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    url :homepage
    regex(/href=.*?8BitDo[._-]Ultimate[._-]Software[._-]V2[._-]macOS[._-]v?(\d+(?:\.\d+)+)\.zip/i)
```

The audit is likely failing due to the livecheck block currently trying to download the whole binary. This should work, and conforms to ours usual standards.